src/App.js

```diff
@@ -1,26 +1,29 @@
-import { BrowserRouter as Router, Routes, Route, NavLink } from 'react-router-dom';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import Home from './component/home';
+import Setting from './component/setting';
 import Detail from './component/detail';
-import Setting from './component/setting'
-// import './App.css';
+import Header from './component/header';
 
 function App() {
   return (
     <Router>
       <div className="App">
-        <header className="App-header">
-          <nav>
-            <NavLink to='/home'>Home</NavLink>
-            <NavLink to='/setting'>Setting</NavLink>
-            <NavLink to='/detail'>Detail</NavLink>
-          </nav>
-        </header>
-          <Routes>
-            <Route path="/home" element={<Home />} />
-            <Route path="/setting" element={<Setting />} />
-            <Route path="/detail" element={<Detail />} />
-            <Route path="/" element={<Home />} />
-          </Routes>
+        <Header />
+        <Routes>
+          <Route index element={<Home />} />
+          <Route path="home" element={<Home />} />
+          <Route path="detail" element={<Detail />} />
+          <Route path="setting" element={<Setting />} />
+        </Routes>
```

Comment:
- [ ] I suggest that you work on one specific item on the category page let us say I chose `eyebrow` categories but it has many types of `eyebrows` so I should be able to check one specific type of eyebrow it is supposed to have a route for that specific page with all information of that specific `eyebrow`

Reply:
Hi @DeliceLydia 

I would like to thank you for your suggestion. You are right I could do that. But I prefer is to make each category as a link and each link will concatenate `searching keyword`  to `URL/detail?product_type=category_name`.

**I think I can implement just I like did before because the requirement says**
- Details page
    1. In the details page, the webapp retrieves data from the API to show detailed data in the category.
    2. When the user clicks on the "Back" button (<), the user navigates to the home page.